operators/src/source/ogr_source.rs

```diff
@@ -532,8 +532,6 @@ where
             return Ok(());
         }
 
-        let mut emitted_non_empty_collections = false;
```

Comment:
is this really redundant?

Reply:
:shrug: It seems so and the tests don't fail.

Reply:
der lint sagt doch
> remove the `else` block and move the contents out

das ist so wie du es gemacht hast meiner Meinung nach nicht äquivalent und scheinbar fehlt dann ein Test der das abprüft

Vorschlag:
```
if is_empty && emitted_non_empty_collections {          
      break;
}
emitted_non_empty_collections = true;

```

Reply:
74ace31 

Ihr hattet Recht!